spec/requests/api/v3/authentication_spec.rb

```diff
@@ -421,10 +423,11 @@ def set_basic_auth_header(user, password)
         .to_return(status: 200, body: JWT::JWK::Set.new(jwk_response).export.to_json, headers: {})
     end
     let(:jwk_response) { jwk }
+    let(:user) { create(:user, identity_url: "keycloak:#{token_sub}") }
 
     before do
       create(:oidc_provider, slug: "keycloak")
-      create(:user, identity_url: "keycloak:#{token_sub}")
+      user.save!
```

Comment:
Would the following be sufficient?
```suggestion
      user
```

Reply:
That's technically sufficient, though I usually find it more intention-revealing to call `save!`. It says "Hey I put that here to make sure we actually perform the side effect of saving the user".

I think in the OP codebase it's also more accepted to just use `let!`, maybe I should've done that, though my personal preference is the way I wrote it here.

WDYT? Should I switch for `let!`?

Reply:
I'd prefer your way or just `user` statement rather than `let!`. So, it is fine by me like it is.